### PR TITLE
Fix Re-admining self on test server (with legacy admin config)

### PR DESCRIPTION
**Pull request by moxian**
## What Does This PR Do
You can now readmin self after de-admining self on test server.

Moved the functionality to parse admins.txt to a dedicated function, and call that, instead of reimplementing the config parser each time it's needed (it's only two, but 50% of those happened to runtime on default example config).

## Why It's Good For The Game
bugfix; makes easier to test stuffs on local server

### CODE REVIEW

File: code/modules/admin/admin_ranks.dm [hunk 1]
```diff
@@ -58,6 +58,28 @@ var/list/admin_ranks = list()								//list of all ranks with associated rights
 	load_admins()
 	return 1
 
+/proc/parse_admins_config_file()
+	var/list/admins = new/list()
+	for(var/line in file2list("config/admins.txt"))
+		// strip out comments
+		var/comment_start = findtext(line, "#")
+		if(comment_start > 0)
+			line = copytext(line, comment_start)
```
Comment (farie82) [inline code/modules/admin/admin_ranks.dm]: idea of the comments is to ignore that line. Not to remove the # of the line then continue

Comment (moxian) [reply code/modules/admin/admin_ranks.dm]: The intention was to remove everything after `#`, not everything before... Ooops...
Fixed now, thanks!

**Comment by Fox-McCloud** [comment]
Please fix.

**Comment by Fox-McCloud** [comment]
No response. Closing.

**Pull request closed without merging**